packages/runtime/test/dataViews/requestItems/CreateRelationshipAttributeRequestItemDVO.test.ts

```diff
@@ -87,9 +87,13 @@ beforeAll(async () => {
 afterAll(() => serviceProvider.stop());
 
 beforeEach(async () => {
-    rEventBus.reset();
-    sEventBus.reset();
+    await sEventBus.waitForRunningEventHandlers();
+    await rEventBus.waitForRunningEventHandlers();
```

Comment:
please use Promise.all to wait for those two busses simultaneously 

```suggestion
    await Promise.all([sEventBus.waitForRunningEventHandlers(), rEventBus.waitForRunningEventHandlers()]);
```